kubeconfig

```diff
@@ -0,0 +1,17 @@
+clusters:
```

Comment:
It seems that this was added unintentionally. Could you please remove it?

Reply:
Done, thank you!0